lib/discovery/index.js

```diff
@@ -57,27 +72,109 @@ export class Discovery extends TypedEmitter {
   }
 
   /**
+   * Return the public key for the identity used for discovery
+   *
    * @returns {string}
+   *
+   * @example
+   *
+   * ```js
+   * import { KeyManager } from '@mapeo/crypto'
+   *
+   * // Create discovery instance
+   * const discover = new Discovery({
+   *  identityKeyPair: new KeyManager(
+   *    KeyManager.generateIdentityKey()
+   *  ).getIdentityKeypair(),
+   * })
+   *
+   * console.log(discover.identityPublicKey) // Some hex-encoded string
+   * ```
    */
   get identityPublicKey() {
     return encodeHex(this.#identityKeyPair.publicKey)
   }
 
   /**
+   * Return the list of subscribed topics
+   *
    * @returns {Topic[]}
+   *
+   * @example
+   *
+   * ```js
+   * import { KeyManager } from '@mapeo/crypto'
+   *
+   * const topicName = 'my-topic'
+   * const km = new KeyManager(KeyManager.generateIdentityKey())
+   * const keyPair = km.getHypercoreKeypair(topicName, Buffer.alloc(32, 0))
+   *
+   * const discover = new Discover({
+   *  identityKeyPair: km.getIdentityKeypair()
+   * })
+   *
+   * await discover.ready()
+   *
+   * discover.join(keyPair.identityPublicKey)
+   *
+   * console.log(discover.topics) // prints out `[Topic]`, where `Topic` is a class instance containing information about the topic joined
+   * ```
    */
   get topics() {
     return Array.from(this.#topics.values())
   }
 
   /**
+   * Returns the list of peers that have been connected to
+   *
    * @returns {PeerInfo[]}
+   *
+   * @example
+   *
+   * ```js
+   *
+   * // Create identities...
```

Comment:
It would be useful to list relevant methods for creating identities here and maybe link to their spot in the docs (not sure off the top of my head if that's possible from within example code using typedoc, so maybe this note would be outside the code block).

Reply:
Yeah makes sense! Another idea: not sure how it would look in the generated output but could put a (collapsible?) snippet for an example of creating the identity, as a footnote type of thing that can be referenced from here. Maybe too complicated so probably stick with your idea